spring-ai-client-chat/src/main/java/org/springframework/ai/chat/client/ChatClientAttributes.java

```diff
@@ -33,7 +30,6 @@ public enum ChatClientAttributes {
 	ADVISORS("spring.ai.chat.client.advisors"),
 	@Deprecated // Only for backward compatibility until the next release.
 	CHAT_MODEL("spring.ai.chat.client.model"),
-	@Deprecated // Only for backward compatibility until the next release.
 	OUTPUT_FORMAT("spring.ai.chat.client.output.format"),
```

Comment:
Not a great approach using the advisor context for this, but we might re-evaluate later when considering the new structured output support.